hack/verify-licenses.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: `hack/verify-licenses.sh`.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+kube::golang::verify_go_version
+
+
+# Ensure that we find the binaries we build before anything else.
+export GOBIN="${KUBE_OUTPUT_BINPATH}"
+PATH="${GOBIN}:${PATH}"
+
+
+# Explicitly opt into go modules, even though we're inside a GOPATH directory
+export GO111MODULE=on
+
+
+# Color Codes
+RED="\033[0;31m"
+GREEN="\033[0;32m"
+YELLOW="\033[0;33m"
+CAYAN="\033[0;36m"
+GRAY="\033[0;37m"
+NC="\033[0m" # No Color
+
+
+declare -a packages_flagged
+declare -a allowed_licenses
+
+LICENSES_JSON="/tmp/$(cd /tmp && mktemp licenses_json.XXXXXX)"
```

Comment:
looks like the best practice is to run `kube::util::ensure-temp-dir` and then use `KUBE_TEMP`, can you please switch to that instead of 3 here?:
https://cs.k8s.io/?q=kube%3A%3Autil%3A%3Aensure-temp-dir&i=nope&files=hack%2F&excludeFiles=&repos=

Reply:
ack. Implemented it the ^ way in the script now.

Thank you for pointing!